pkgs/development/libraries/libzip/default.nix

```
@@ -4,14 +4,21 @@
 , fetchurl
 , perl
 , zlib
+, groff
+, withBzip2 ? false
+, bzip2
+, withLZMA ? false
+, lzma
```

```suggestion
, xz
```

lzma is an alias to xz which is going to be removed soon.